examples/pytorch_distributed/pytorch_distributed.py

```diff
@@ -0,0 +1,81 @@
+# A basic example showing how to use Runhouse to run a PyTorch distributed training script on a cluster of GPUs.
+
+import asyncio
+
+import runhouse as rh
+import torch
+
+from torch.nn.parallel import DistributedDataParallel as DDP
+
+
+# Define the PyTorch distributed training script
+def train():
+    # Initialize the distributed training environment,
+    # per https://pytorch.org/docs/stable/distributed.html#initialization
+    torch.distributed.init_process_group(backend="nccl")
+    rank = torch.distributed.get_rank()
+    print(f"Rank {rank} of {torch.distributed.get_world_size()} initialized")
+
+    # Create a simple model and optimizer
+    device_id = rank % torch.cuda.device_count()
+    model = torch.nn.Linear(10, 1).cuda()
+    model = DDP(model, device_ids=[device_id])
+    optimizer = torch.optim.SGD(model.parameters(), lr=0.01)
+
+    # Perform a simple training loop
+    for epoch in range(10):
+        optimizer.zero_grad()
+        output = model(torch.randn(10).cuda())
+        loss = output.sum()
+        loss.backward()
+        optimizer.step()
+
+        print(f"Rank {rank}: Epoch {epoch}, Loss {loss.item()}")
+
+    # Clean up the distributed training environment
+    torch.distributed.destroy_process_group()
+
+
+async def main():
+    # Create a cluster of 2 GPUs
```

Comment:
I think removing this in favor of a longer intro text above `async def main()` will render much better:

```python
# ## Define workers and run training on a remote cluster
# Now, we define the main function that will run locally when we run
# this script, and set up our Runhouse module on a remote cluster.
async def main():
    ...
```